.github/actions/spelling/line_forbidden.patterns

```diff
@@ -1,119 +1,289 @@
 # reject `m_data` as VxWorks defined it and that breaks things if it's used elsewhere
 # see [fprime](https://github.com/nasa/fprime/commit/d589f0a25c59ea9a800d851ea84c2f5df02fb529)
 # and [Qt](https://github.com/qtproject/qt-solutions/blame/fb7bc42bfcc578ff3fa3b9ca21a41e96eb37c1c7/qtscriptclassic/src/qscriptbuffer_p.h#L46)
-# \bm_data\b
+#\bm_data\b
 
+# Were you debugging using a framework with `fit()`?
 # If you have a framework that uses `it()` for testing and `fit()` for debugging a specific test,
-# you might not want to check in code where you were debugging w/ `fit()`, in which case, you might want
-# to use this:
+# you might not want to check in code where you skip all the other tests.
 #\bfit\(
 
-# s.b. anymore
+# Should be `HH:MM:SS`
+\bHH:SS:MM\b
+
+# Should be `86400` (seconds in a standard day)
```

Comment:
this is such an excellent use of tooling :)

Reply:
I believe this and some of the others near it in this file were suggestions from others. If you know of others that are common mistakes, I'm definitely open to adding them to spell-check-this. (And you can certainly add your own for this repository.)